### ISSUE TITLE
a redacted crash report (build exec )

### ISSUE THREAD

**Issue by zsj1029** [oven-sh/bun#14489]
### Relevant log output

```
Bun v1.1.30 (7996d06b) Windows x64
Windows v.win10_fe
CPU: sse42 avx avx2
Args: "C:\Users\zsj10\.bun\bin\bun.exe" "build" "--minify" "--sourcemap" "./src/index.ts" "--target=bun-linux-x64" "--compile" "--outfile" "./dist/cephfs"
Features: define tsconfig(20) 
Elapsed: 143ms | User: 31ms | Sys: 78ms
RSS: 0.15GB | Peak: 0.15GB | Commit: 0.26GB | Faults: 39025

panic(thread 38960): Expected pretty file path to have only forward slashes, got 'C:\Users\zsj10\Desktop\cephfs-node-service\node_modules\.pnpm\body-parser@1.20.3\node_modules\bytes\index.js'
oh no: Bun has crashed. This indicates a bug in Bun, not your code.
```

### Stack Trace (bun.report)

Bun v1.1.30 ([`7996d06`](<https://github.com/oven-sh/bun/tree/7996d06b8f5869ccc9ad9a421cd06469da4273fc>)) on windows x86_64 [BuildCommand]

**panic**: Expected pretty file path to have only forward slashes, got 'H:\GitHub\clapper\node_modules\@aitube\clap\dist\index.js'

- [`fs.zig:1728`](<https://github.com/oven-sh/bun/blob/7996d06b8f5869ccc9ad9a421cd06469da4273fc/src/fs.zig#L1728>): `assertPrettyIsValid`
- [`bundle_v2.zig:9002`](<https://github.com/oven-sh/bun/blob/7996d06b8f5869ccc9ad9a421cd06469da4273fc/src/bundler/bundle_v2.zig#L9002>): `postProcessJSChunk`
- [`bundle_v2.zig:7904`](<https://github.com/oven-sh/bun/blob/7996d06b8f5869ccc9ad9a421cd06469da4273fc/src/bundler/bundle_v2.zig#L7904>): `generateChunk`
- [`atomic.zig:40`](<https://github.com/oven-sh/bun/blob/7996d06b8f5869ccc9ad9a421cd06469da4273fc//opt/homebrew/etc/buildkite-agent/builds/darwin-aarch64-studio-2-lf5iax/bun/bun/vendor/zig/lib/std/atomic.zig#L40>): `load`

Features: external, tsconfig, tsconfig\_paths, tsconfig\_paths

**Comment by paperdave** [comment]
fixed by #11430

**Comment by zsj1029** [comment]
[#14767](https://github.com/oven-sh/bun/issues/14767)
